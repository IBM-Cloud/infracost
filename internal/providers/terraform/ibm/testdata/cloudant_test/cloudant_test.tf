terraform {
  required_providers {
    ibm = {
      source  = "IBM-Cloud/ibm"
      version = "1.42.0-beta0"
    }
  }
}

provider "ibm" {
  # Configuration options
  region = "us-south"
}

resource "ibm_cloudant" "cloudant_lite" {
  name     = "cloudant_prt"
  location = "us-south"
  plan     = "lite"
  capacity = 1

  legacy_credentials  = true
  include_data_events = false
  enable_cors         = true

  cors_config {
    allow_credentials = false
    origins           = ["https://example.com"]
  }
}

resource "ibm_cloudant" "cloudant_std" {
  name     = "cloudant_prt"
  location = "us-south"
  plan     = "standard"
  capacity = 1

  legacy_credentials  = true
  include_data_events = false
  enable_cors         = true

  cors_config {
    allow_credentials = false
    origins           = ["https://example.com"]
  }
}

resource "ibm_cloudant" "with_usage" {
  name     = "cloudant_prt"
  location = "us-south"
  plan     = "standard"
  capacity = 2

  legacy_credentials  = true
  include_data_events = false
  enable_cors         = true

  cors_config {
    allow_credentials = false
    origins           = ["https://example.com"]
  }
}



