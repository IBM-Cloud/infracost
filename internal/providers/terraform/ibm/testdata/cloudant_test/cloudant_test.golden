
 Name                                    Monthly Qty  Unit            Monthly Cost 
                                                                                   
 Cloudant - Dedicated-Hardware                                                     
 └─ Dedicated Hardware                             1  instance           $5,000.00 
                                                                                   
 Cloudant - Lite                                                                   
 └─ Cloudant                                       1  instance               $0.00 
                                                                                   
 Cloudant - Standard                                                               
 ├─ Monthly Reads                                100  reads/second          $25.00 
 ├─ Monthly Writes                                50  writes/second         $25.00 
 ├─ Monthly Global Queries                         5  queries/second        $25.00 
 └─ Free Estimated Storage (first 20GB)           20  GB                     $0.00 
                                                                                   
 Cloudant - Standard                                                               
 ├─ Monthly Reads                                100  reads/second          $25.00 
 ├─ Monthly Writes                                50  writes/second         $25.00 
 ├─ Monthly Global Queries                         5  queries/second        $25.00 
 ├─ Free Estimated Storage (first 20GB)           20  GB                     $0.00 
 └─ Estimated Storage                             20  GB                    $20.00 
                                                                                   
 OVERALL TOTAL                                                           $5,170.00 
──────────────────────────────────
4 cloud resources were detected:
∙ 4 were estimated